src/libxrpl/protocol/STParsedJSON.cpp

```diff
@@ -398,8 +398,16 @@ parseLeaf(
 
                     std::uint64_t val;
 
+                    bool const useBase10 = field == sfMaximumAmount ||
```

Comment:
Same - generic type has knowledge of specific fields.

Reply:
ditto